PageObjects/APII/LoginAPI.js

```diff
@@ -0,0 +1,69 @@
+import endpoints from "../../Fixtures/endpoints.json";
+import dotenv from "dotenv";
+import path from "path";
+import { expect } from "@playwright/test";
+dotenv.config({ path: path.resolve(__dirname, "../.env") });
+
+export class LoginAPI {
+  constructor(request) {
+    (this.baseUrl = process.env.BASEURLAPI), (this.request = request);
+    this.successLoginMessage = "User logged in successfully";
+  }
+  async getToken({
```

Comment:
you need only 1 function for login that will cover all things :
- check status code (status code should be a param of function)
- check token
- return token

You need code, if status 200 return token if other, return errors etc... but it should all be part of 1 function, not 3